x-pack/distributions/internal/observabilitySRE/build-ext.gradle

```diff
@@ -28,4 +28,20 @@ allprojects {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+project(':logstash-core') {
+    afterEvaluate {
+        if (rootProject.fedrampHighMode) {
+            logger.lifecycle("Adding BouncyCastle FIPS dependencies to logstash-core")
+            dependencies {
+                // Add FIPS dependencies to the runtimeClasspath configuration
+                // This ensures they'll be included by the existing copyRuntimeLibs task
+                runtimeClasspath "org.bouncycastle:bc-fips:2.0.0"
+                runtimeClasspath "org.bouncycastle:bcpkix-fips:2.0.7"
+                runtimeClasspath "org.bouncycastle:bctls-fips:2.0.19"
+                runtimeClasspath "org.bouncycastle:bcutil-fips:2.0.3"
```

Comment:
🤔 since gradle [doesn't like us declaring dependencies against the `runtimeClasspath`](https://buildkite.com/elastic/logstash-pull-request-pipeline/builds/2661#0195b022-2b41-4519-b595-f1ca25d9823c/6-65), could we define a stand-alone configuration for them to be declared against, and make `runtimeOnly` inherit from it if in `fedRampHighMode`? I _think_ this will make the resolved `runtimeClasspath` include it, which is good for both running tests and for the `copyRuntimeLibs` task requirements.